src/Sylius/Component/Translation/Provider/TranslationLocaleProvider.php

```diff
@@ -9,10 +9,9 @@
 * file that was distributed with this source code.
 */
 
-namespace Sylius\Bundle\CoreBundle\Locale;
+namespace Sylius\Component\Translation\Provider;
 
 use Sylius\Component\Locale\Context\LocaleContextInterface;
```

Comment:
The `Translation` component shouldn't be coupled to `Locale` component. This class should be moved to `Core` component and after the `Translation` is merged into `Resource` component (#3750) we can move it to `Locale`.